apps/web/src/pages/integrations/components/multi-provider/SelectProviderSidebar.tsx

```diff
@@ -0,0 +1,303 @@
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { ActionIcon, ColorScheme, Group, Space, Stack, Tabs, TabsValue, useMantineColorScheme } from '@mantine/core';
+import { ChannelTypeEnum, providers } from '@novu/shared';
+import { CONTEXT_PATH } from '../../../../config';
+import { colors } from '../../../../design-system';
+import { useDebounce } from '../../../../hooks';
+import { Button, Input, Title, Tooltip, Text } from '../../../../design-system';
+import { Search, Close } from '../../../../design-system/icons';
+import { ChannelTitle } from '../../../templates/components/ChannelTitle';
+import useStyles from '../../../../design-system/tabs/Tabs.styles';
+import { IIntegratedProvider } from '../../IntegrationsStoreModal';
+import { getGradient } from '../../../../design-system/config/helper';
+import { useNavigate } from 'react-router-dom';
+import { CHANNELS_ORDER } from '../IntegrationsListNoData';
+import { CHANNEL_TYPE_TO_STRING } from '../../../../utils/channels';
+
+const mapStructure = (listProv): IIntegratedProvider[] =>
+  listProv.map((providerItem) => ({
+    providerId: providerItem.id,
+    displayName: providerItem.displayName,
+    channel: providerItem.channel,
+    docReference: providerItem.docReference,
+  }));
+
+const getLogoFileName = (id, schema: ColorScheme): string => {
+  if (schema === 'dark') {
+    return `${CONTEXT_PATH}/static/images/providers/dark/square/${id}.svg`;
+  }
+
+  return `${CONTEXT_PATH}/static/images/providers/light/square/${id}.svg`;
+};
+
+export function SelectProviderSidebar() {
+  const [{ emailProviders, smsProviders, chatProviders, pushProviders, inAppProviders }, setProviders] = useState({
+    emailProviders: mapStructure(
+      providers.filter((providerItem) => providerItem.channel === ChannelTypeEnum.EMAIL) || []
```

Comment:
each channel providers could be imported separately like:
```
import { emailProviders, smsProviders, ... } from '@novu/shared';
```
then you won't need to filter